addons/circulation/stringtable.xml

```diff
@@ -2089,5 +2089,9 @@
             <German>Alternativer Ton</German>
             <Czech>Střídavý tón</Czech>
         </Key>
+        <Key ID="STR_KAT_Circulation_SETTING_Threshold_SpO2Warning">
+            <English>AED-X Alarm SpO2 Threshold</English>
```

Comment:
alarm is written in lower case same as threshold

Reply:
The same issue is in the original string I copied from, but I'll fix it here 😅.